src/scripts/pod-sweeper.sh

```diff
@@ -0,0 +1,69 @@
+#!/bin/bash
+# Copyright 2024 Canonical Ltd.
+# See LICENSE file for licensing details.
+
+# https://github.com/airbytehq/airbyte-platform/blob/main/charts/airbyte-pod-sweeper/templates/configmap.yaml
+# TODO(kelkawi-a): Move this to Airbyte ROCK
+
+get_job_pods() {
+    # echo "Running kubectl command to get job pods..."
+    kubectl -n "${JOB_KUBE_NAMESPACE}" -L airbyte -l airbyte=job-pod \
+        get pods \
+        -o=jsonpath='{range .items[*]} {.metadata.name} {.status.phase} {.status.conditions[0].lastTransitionTime} {.status.startTime}{"\n"}{end}'
+}
+
+delete_pod() {
+    printf "From status '%s' since '%s', " "$2" "$3"
+    echo "$1" | grep -v "STATUS" | awk '{print $1}' | xargs --no-run-if-empty kubectl -n "${JOB_KUBE_NAMESPACE}" delete pod
+}
```

Comment:
Airbyte itself creates k8s pods for different sync jobs. This script is meant to periodically clean up job pods which have exceeded a certain ttl value. I've added a note to the README to reflect that Airbyte creates these pods. 

Scaling with this charm has not been tested yet, but this might be modified depending on the outcome of such tests.